plugin/Makefile

```diff
@@ -17,37 +17,65 @@
 # Eventually we will throw away deb/rpm build from ../vmdk_plugin/Makefile and will use only
 # managed plugin
 
-# Allow user name for dockerhub (and first components of the plugin name) to be supplied in env
-# Otherwise, grab it from current user (`whoami`)
-DOCKER_HUB_USER ?= $(shell whoami)
+# Makefile by default build a name in this format (example) 'msterin/docker-volume-vsphere:0.13-dev'
+#
+# You can use the following environment vars to change the behavior :
+#
+# DOCKER_HUB_USER - name of user and dockerhub repo to use. Default is `whoami`
+#                   Please 'docker login' there is you want to push
```

Comment:
It was not clear earlier with ```DOCKER_HUB_USER``` and ```Please 'docker login' there is you want to push``` .. after renaming it to ```docker_hub_repo``` it makes sense that ```DOCKER_HUB_USER``` needs to be set as environment explicitly if you are planning to push ```<docker_user_ids>/docker-volume-vsphere``` or ```cnastorage/docker-volume-vsphere``` as many of us are not having docker_user as `whoami`

No issues now as the variable got renamed now.